python/py-rangehttpserver/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        danvk RangeHTTPServer 1.2.0
+name                py-${github.project}
```

Comment:
this should become ```name py-rangehttpserver``` without the capitalization to avoid an error when doing ```port lint --nitpick```: ```Error: Portfile directory py-rangehttpserver does not match port name py-RangeHTTPServer```. (Please do run the lint command, before submitting the PR as you say you did in the checklist).

Reply:
I run the lint command at the first submission, that I did not again while modifying. I'll do that from now on, and sorry.

Reply:
no, don't change the ```github.setup```... as I said you should change the ```name``` line to:
```name py-rangehttpserver```